libraries/botbuilder/lib/recognizerResult.d.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * @module botbuilder
+ */
+/** second comment block */
+/**
+ * A result from a recognizer
+ */
+export interface RecognizerResult {
+    text: string;
+    intents: any;
+    entities: any;
+    $instance?: any;
```

Comment:
Hmm.  We could use this for meta-data about the whole result, but it would not be the most likely place right now.

Reply:
Yep, it's a residual from the old approach, removing.